ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/privacy/PrivateTransactionHandler.java

```diff
@@ -56,7 +57,9 @@ public PrivateTransactionHandler(
     this.nodeKeyPair = nodeKeyPair;
   }
 
-  public Transaction handle(final PrivateTransaction privateTransaction) throws IOException {
+  public Transaction handle(
+      final PrivateTransaction privateTransaction, final Supplier<Long> nonceSupplier)
+      throws IOException {
     LOG.trace("Handling private transaction");
```

Comment:
Should we log the transaction in this trace message?

Reply:
sounds sensible